Hunting Queries/SecurityEvent/Crashdumpdisabledonhost.yaml

```diff
@@ -0,0 +1,30 @@
+id: 5a3615af-21c9-427e-8bf1-ed2350992bb4
+name: Crash dump disabled on host
+description: |
+  'This detection looks the prevention of crash dumps being created. This can be used to limit reporting by malware, look for suspicious processes setting this registry key.'
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvents
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1070
+query: |
+  SecurityEvent
```

Comment:
Why are we duplicating this when we already have imRegistry above?

Reply:
Neither imRegistry or imProcess and built in parsers. There will be a large number of workspaces that can't currently leverage them directly so providing both options for greatest reach and impact.